### PR TITLE
[loki-canary] Adding flexibility to add pod labels to loki-canary daemon set

### PR DESCRIPTION
**Pull request by praveenjindal62**
We are using loki-canary charts and wanted to manage pods using some custom labels on pods. Loki-Canary doesn't provide fexibility yet to use custom podLabels on Daemonset loki-canary. Added .Values.podLabels that can be applied to loki-canary daemonset.

### CODE REVIEW

File: charts/loki-canary/values.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ priorityClassName: null
 # -- Common annotations for all pods
 podAnnotations: {}
 
+# -- Common labels for all pods
+podLabels: {}
```
Comment (trevorwhitney) [inline charts/loki-canary/values.yaml:25]: WDYT of making this `selectorLabels` or `podSelectorLabels` to match the convention in the helper function `loki-canary.selectorLabels` well call right before including this?

Comment (trevorwhitney) [inline charts/loki-canary/values.yaml:25]: WDYT of making this `selectorLabels` or `podSelectorLabels` to match the convention in the helper function `loki-canary.selectorLabels` well call right before including this?

**Review by trevorwhitney — COMMENTED** [review]
Looks like a good addition, thank you! One small nit-picky question.

File: charts/loki-canary/values.yaml (hunk 1 above)
Comment (trevorwhitney) [reply charts/loki-canary/values.yaml:25]: ahh, so you want to differentiate these from `selectorLabels` as in they are _only_ meant for the pod. I agree with that argument

**Review by trevorwhitney — APPROVED** [review]
LGTM

**Comment by praveenjindal62** [comment]
Hi @trevorwhitney , Could you please approve it again, I had to update README.md via helm-docs which I didn't do earlier so one of the workflow was failing

**Comment by praveenjindal62** [comment]
@trevorwhitney  @unguiculus  @Whyeasy Please approve this merge request

**Pull request merged**